src/AuditingServiceProvider.php

```diff
@@ -26,7 +27,7 @@ public function boot()
         $this->mergeConfigFrom(__DIR__.'/../config/audit.php', 'audit');
 
         Event::listen(AuditCustom::class, RecordCustomAudit::class);
-        Event::listen(DispatchAudit::class, ProcessDispatchAudit::class);
+        Event::listen(DispatchAudit::class, Config::string('audit.queue.listener', ProcessDispatchAudit::class));
```

Comment:
@willpower232 hi, maybe an interface check, to prevent someone from setting up a class that is not a listener, or some non-existent class

Reply:
It is a good option, another could be to create a new contract that requires having a `handle` method and validate that interface.